examples/web-ui/material/menus/menuDesktop.smash

```diff
@@ -0,0 +1,9 @@
+* Set Desktop
+    * Click first level page button {{page}}
```

Comment:
Not sure what you mean by "first level", and the word {{page}} doesn't seem to fit in here grammatically. Totally valid code, but the examples should be really polished up imo.